mne/viz/ica.py

```diff
@@ -234,8 +234,8 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
 
 def _get_psd_label_and_std(this_psd, dB, ica, num_std):
     """Handle setting up PSD for one component, for plot_ica_properties."""
-    psd_ylabel = _convert_psds(this_psd, dB, estimate='auto', scaling=1.,
-                               unit='AU', ch_names=ica.ch_names)
+    psd_ylabel = _convert_psds(this_psd[:, :], dB, estimate='auto', scaling=1.,
+                               unit='AU', first_dim='epoch')
```

Comment:
why did you have  to add [:, :] and not pass ch_names?

Reply:
oops, that was left over from my playing around while debugging. I've reverted that change.

Reply:
`ch_names` I don't have to pass, because they are only used to generate the error message. The first dimensions of `this_psd` are epochs x freqs in the case you see here. No channels involved.